monitoring_status/controllers/main.py

```diff
@@ -31,10 +34,24 @@ class Monitoring(http.Controller):
     @http.route('/monitoring/status', type='http', auth='none')
     def status(self):
         ensure_db()
+        http_status = 200
         # TODO: add 'sub-systems' status and infos:
         # queue job, cron, database, ...
```

Comment:
I guess better to keep database from this todo list, as this is about status and not infos ?